administration/intro.adoc

```diff
@@ -332,4 +332,55 @@ to fix it, delete manually finalizer from the resource.
 ----
 $ kubectl -n kubevirt patch kv kubevirt --type=json -p '[{ "op": "remove", "path": "/metadata/finalizers" }]'
 ----
+
+Note: If you find your namespace is still hanging in terminationstate this script will delete all objects inside of a namespace.
```

Comment:
this does not sound very safe. can you add a note why this is needed? how long should we typically wait for the normal cleanup to finish?

Reply:
This is needed to clean up all kubernetes objects when uninstall does not go as expected. This usually happens when things are not cleaned up in the correct order, or race conditions happen.